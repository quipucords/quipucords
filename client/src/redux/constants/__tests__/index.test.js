import reduxTypes, * as allReduxTypes from '..';

describe('ReduxTypes', () => {
  it('should have specific type properties', () => {
    expect(reduxTypes).toMatchSnapshot('specific types');

    expect(allReduxTypes).toMatchSnapshot('all redux types');
  });
});
