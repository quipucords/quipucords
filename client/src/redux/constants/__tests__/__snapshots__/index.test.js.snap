// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all redux types 1`] = `
Object {
  "confirmationModalTypes": Object {
    "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
    "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
  },
  "credentialsTypes": Object {
    "ADD_CREDENTIAL": "ADD_CREDENTIAL",
    "CREATE_CREDENTIAL_SHOW": "CREATE_CREDENTIAL_SHOW",
    "DELETE_CREDENTIAL": "DELETE_CREDENTIAL",
    "DELETE_CREDENTIALS": "DELETE_CREDENTIALS",
    "EDIT_CREDENTIAL_SHOW": "EDIT_CREDENTIAL_SHOW",
    "GET_CREDENTIAL": "GET_CREDENTIAL",
    "GET_CREDENTIALS": "GET_CREDENTIALS",
    "GET_WIZARD_CREDENTIALS": "GET_WIZARD_CREDENTIALS",
    "RESET_CREDENTIAL_UPDATE_STATUS": "RESET_CREDENTIAL_UPDATE_STATUS",
    "UPDATE_CREDENTIAL": "UPDATE_CREDENTIAL",
    "UPDATE_CREDENTIAL_HIDE": "UPDATE_CREDENTIAL_HIDE",
  },
  "default": Object {
    "confirmationModal": Object {
      "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
      "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
    },
    "credentials": Object {
      "ADD_CREDENTIAL": "ADD_CREDENTIAL",
      "CREATE_CREDENTIAL_SHOW": "CREATE_CREDENTIAL_SHOW",
      "DELETE_CREDENTIAL": "DELETE_CREDENTIAL",
      "DELETE_CREDENTIALS": "DELETE_CREDENTIALS",
      "EDIT_CREDENTIAL_SHOW": "EDIT_CREDENTIAL_SHOW",
      "GET_CREDENTIAL": "GET_CREDENTIAL",
      "GET_CREDENTIALS": "GET_CREDENTIALS",
      "GET_WIZARD_CREDENTIALS": "GET_WIZARD_CREDENTIALS",
      "RESET_CREDENTIAL_UPDATE_STATUS": "RESET_CREDENTIAL_UPDATE_STATUS",
      "UPDATE_CREDENTIAL": "UPDATE_CREDENTIAL",
      "UPDATE_CREDENTIAL_HIDE": "UPDATE_CREDENTIAL_HIDE",
    },
    "facts": Object {
      "ADD_FACTS": "ADD_FACTS",
      "default": "ADD_FACTS",
    },
    "reports": Object {
      "GET_MERGE_REPORT": "GET_MERGE_REPORT",
      "GET_REPORT": "GET_REPORT",
    },
    "scans": Object {
      "ADD_SCAN": "ADD_SCAN",
      "CANCEL_SCAN": "CANCEL_SCAN",
      "DELETE_SCAN": "DELETE_SCAN",
      "GET_MERGE_SCAN_RESULTS": "GET_MERGE_SCAN_RESULTS",
      "GET_SCAN": "GET_SCAN",
      "GET_SCANS": "GET_SCANS",
      "GET_SCAN_CONNECTION_RESULTS": "GET_SCAN_CONNECTION_RESULTS",
      "GET_SCAN_INSPECTION_RESULTS": "GET_SCAN_INSPECTION_RESULTS",
      "GET_SCAN_JOB": "GET_SCAN_JOB",
      "GET_SCAN_JOBS": "GET_SCAN_JOBS",
      "MERGE_SCAN_DIALOG_HIDE": "MERGE_SCAN_DIALOG_HIDE",
      "MERGE_SCAN_DIALOG_SHOW": "MERGE_SCAN_DIALOG_SHOW",
      "PAUSE_SCAN": "PAUSE_SCAN",
      "RESET_SCAN_ADD_STATUS": "RESET_SCAN_ADD_STATUS",
      "RESTART_SCAN": "RESTART_SCAN",
      "START_SCAN": "START_SCAN",
      "UPDATE_SCAN": "UPDATE_SCAN",
    },
    "sources": Object {
      "ADD_SOURCE": "ADD_SOURCE",
      "CREATE_SOURCE_SHOW": "CREATE_SOURCE_SHOW",
      "DELETE_SOURCE": "DELETE_SOURCE",
      "DELETE_SOURCES": "DELETES_SOURCE",
      "EDIT_SOURCE_SHOW": "EDIT_SOURCE_SHOW",
      "GET_SCANS_SOURCES": "GET_SCANS_SOURCES",
      "GET_SOURCE": "GET_SOURCE",
      "GET_SOURCES": "GET_SOURCES",
      "INVALID_SOURCE_WIZARD_STEPTWO": "INVALID_SOURCE_WIZARD_STEPTWO",
      "UPDATE_SOURCE": "UPDATE_SOURCE",
      "UPDATE_SOURCE_HIDE": "UPDATE_SOURCE_HIDE",
      "UPDATE_SOURCE_WIZARD": "UPDATE_SOURCE_WIZARD",
      "UPDATE_SOURCE_WIZARD_STEPONE": "UPDATE_SOURCE_WIZARD_STEPONE",
      "UPDATE_SOURCE_WIZARD_STEPTWO": "UPDATE_SOURCE_WIZARD_STEPTWO",
    },
    "status": Object {
      "STATUS_INFO": "STATUS_INFO",
      "default": "STATUS_INFO",
    },
    "toastNotifications": Object {
      "TOAST_ADD": "TOAST_ADD",
      "TOAST_PAUSE": "TOAST_PAUSE",
      "TOAST_REMOVE": "TOAST_REMOVE",
      "TOAST_RESUME": "TOAST_RESUME",
    },
    "user": Object {
      "USER_AUTH": "USER_AUTH",
      "USER_INFO": "USER_INFO",
      "USER_LOGOUT": "USER_LOGOUT",
    },
    "view": Object {
      "CREDENTIALS_VIEW": "CREDENTIALS_VIEW",
      "DESELECT_ITEM": "DESELECT_ITEM",
      "EXPAND_ITEM": "EXPAND_ITEM",
      "SCANS_VIEW": "SCANS_VIEW",
      "SELECT_ITEM": "SELECT_ITEM",
      "SOURCES_VIEW": "SOURCES_VIEW",
    },
    "viewPagination": Object {
      "SET_PER_PAGE": "SET_PER_PAGE",
      "VIEW_FIRST_PAGE": "VIEW_FIRST_PAGE",
      "VIEW_LAST_PAGE": "VIEW_LAST_PAGE",
      "VIEW_NEXT_PAGE": "VIEW_NEXT_PAGE",
      "VIEW_PAGE_NUMBER": "VIEW_PAGE_NUMBER",
      "VIEW_PREVIOUS_PAGE": "VIEW_PREVIOUS_PAGE",
    },
    "viewToolbar": Object {
      "ADD_FILTER": "ADD_FILTER",
      "CLEAR_FILTERS": "CLEAR_FILTERS",
      "REMOVE_FILTER": "REMOVE_FILTER",
      "SET_FILTER_TYPE": "SET_FILTER_TYPE",
      "SET_FILTER_VALUE": "SET_FILTER_VALUE",
      "SET_SORT_TYPE": "SET_SORT_TYPE",
      "TOGGLE_SORT_ASCENDING": "TOGGLE_SORT_ASCENDING",
    },
  },
  "factsTypes": Object {
    "ADD_FACTS": "ADD_FACTS",
    "default": "ADD_FACTS",
  },
  "reduxTypes": Object {
    "confirmationModal": Object {
      "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
      "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
    },
    "credentials": Object {
      "ADD_CREDENTIAL": "ADD_CREDENTIAL",
      "CREATE_CREDENTIAL_SHOW": "CREATE_CREDENTIAL_SHOW",
      "DELETE_CREDENTIAL": "DELETE_CREDENTIAL",
      "DELETE_CREDENTIALS": "DELETE_CREDENTIALS",
      "EDIT_CREDENTIAL_SHOW": "EDIT_CREDENTIAL_SHOW",
      "GET_CREDENTIAL": "GET_CREDENTIAL",
      "GET_CREDENTIALS": "GET_CREDENTIALS",
      "GET_WIZARD_CREDENTIALS": "GET_WIZARD_CREDENTIALS",
      "RESET_CREDENTIAL_UPDATE_STATUS": "RESET_CREDENTIAL_UPDATE_STATUS",
      "UPDATE_CREDENTIAL": "UPDATE_CREDENTIAL",
      "UPDATE_CREDENTIAL_HIDE": "UPDATE_CREDENTIAL_HIDE",
    },
    "facts": Object {
      "ADD_FACTS": "ADD_FACTS",
      "default": "ADD_FACTS",
    },
    "reports": Object {
      "GET_MERGE_REPORT": "GET_MERGE_REPORT",
      "GET_REPORT": "GET_REPORT",
    },
    "scans": Object {
      "ADD_SCAN": "ADD_SCAN",
      "CANCEL_SCAN": "CANCEL_SCAN",
      "DELETE_SCAN": "DELETE_SCAN",
      "GET_MERGE_SCAN_RESULTS": "GET_MERGE_SCAN_RESULTS",
      "GET_SCAN": "GET_SCAN",
      "GET_SCANS": "GET_SCANS",
      "GET_SCAN_CONNECTION_RESULTS": "GET_SCAN_CONNECTION_RESULTS",
      "GET_SCAN_INSPECTION_RESULTS": "GET_SCAN_INSPECTION_RESULTS",
      "GET_SCAN_JOB": "GET_SCAN_JOB",
      "GET_SCAN_JOBS": "GET_SCAN_JOBS",
      "MERGE_SCAN_DIALOG_HIDE": "MERGE_SCAN_DIALOG_HIDE",
      "MERGE_SCAN_DIALOG_SHOW": "MERGE_SCAN_DIALOG_SHOW",
      "PAUSE_SCAN": "PAUSE_SCAN",
      "RESET_SCAN_ADD_STATUS": "RESET_SCAN_ADD_STATUS",
      "RESTART_SCAN": "RESTART_SCAN",
      "START_SCAN": "START_SCAN",
      "UPDATE_SCAN": "UPDATE_SCAN",
    },
    "sources": Object {
      "ADD_SOURCE": "ADD_SOURCE",
      "CREATE_SOURCE_SHOW": "CREATE_SOURCE_SHOW",
      "DELETE_SOURCE": "DELETE_SOURCE",
      "DELETE_SOURCES": "DELETES_SOURCE",
      "EDIT_SOURCE_SHOW": "EDIT_SOURCE_SHOW",
      "GET_SCANS_SOURCES": "GET_SCANS_SOURCES",
      "GET_SOURCE": "GET_SOURCE",
      "GET_SOURCES": "GET_SOURCES",
      "INVALID_SOURCE_WIZARD_STEPTWO": "INVALID_SOURCE_WIZARD_STEPTWO",
      "UPDATE_SOURCE": "UPDATE_SOURCE",
      "UPDATE_SOURCE_HIDE": "UPDATE_SOURCE_HIDE",
      "UPDATE_SOURCE_WIZARD": "UPDATE_SOURCE_WIZARD",
      "UPDATE_SOURCE_WIZARD_STEPONE": "UPDATE_SOURCE_WIZARD_STEPONE",
      "UPDATE_SOURCE_WIZARD_STEPTWO": "UPDATE_SOURCE_WIZARD_STEPTWO",
    },
    "status": Object {
      "STATUS_INFO": "STATUS_INFO",
      "default": "STATUS_INFO",
    },
    "toastNotifications": Object {
      "TOAST_ADD": "TOAST_ADD",
      "TOAST_PAUSE": "TOAST_PAUSE",
      "TOAST_REMOVE": "TOAST_REMOVE",
      "TOAST_RESUME": "TOAST_RESUME",
    },
    "user": Object {
      "USER_AUTH": "USER_AUTH",
      "USER_INFO": "USER_INFO",
      "USER_LOGOUT": "USER_LOGOUT",
    },
    "view": Object {
      "CREDENTIALS_VIEW": "CREDENTIALS_VIEW",
      "DESELECT_ITEM": "DESELECT_ITEM",
      "EXPAND_ITEM": "EXPAND_ITEM",
      "SCANS_VIEW": "SCANS_VIEW",
      "SELECT_ITEM": "SELECT_ITEM",
      "SOURCES_VIEW": "SOURCES_VIEW",
    },
    "viewPagination": Object {
      "SET_PER_PAGE": "SET_PER_PAGE",
      "VIEW_FIRST_PAGE": "VIEW_FIRST_PAGE",
      "VIEW_LAST_PAGE": "VIEW_LAST_PAGE",
      "VIEW_NEXT_PAGE": "VIEW_NEXT_PAGE",
      "VIEW_PAGE_NUMBER": "VIEW_PAGE_NUMBER",
      "VIEW_PREVIOUS_PAGE": "VIEW_PREVIOUS_PAGE",
    },
    "viewToolbar": Object {
      "ADD_FILTER": "ADD_FILTER",
      "CLEAR_FILTERS": "CLEAR_FILTERS",
      "REMOVE_FILTER": "REMOVE_FILTER",
      "SET_FILTER_TYPE": "SET_FILTER_TYPE",
      "SET_FILTER_VALUE": "SET_FILTER_VALUE",
      "SET_SORT_TYPE": "SET_SORT_TYPE",
      "TOGGLE_SORT_ASCENDING": "TOGGLE_SORT_ASCENDING",
    },
  },
  "reportsTypes": Object {
    "GET_MERGE_REPORT": "GET_MERGE_REPORT",
    "GET_REPORT": "GET_REPORT",
  },
  "scansTypes": Object {
    "ADD_SCAN": "ADD_SCAN",
    "CANCEL_SCAN": "CANCEL_SCAN",
    "DELETE_SCAN": "DELETE_SCAN",
    "GET_MERGE_SCAN_RESULTS": "GET_MERGE_SCAN_RESULTS",
    "GET_SCAN": "GET_SCAN",
    "GET_SCANS": "GET_SCANS",
    "GET_SCAN_CONNECTION_RESULTS": "GET_SCAN_CONNECTION_RESULTS",
    "GET_SCAN_INSPECTION_RESULTS": "GET_SCAN_INSPECTION_RESULTS",
    "GET_SCAN_JOB": "GET_SCAN_JOB",
    "GET_SCAN_JOBS": "GET_SCAN_JOBS",
    "MERGE_SCAN_DIALOG_HIDE": "MERGE_SCAN_DIALOG_HIDE",
    "MERGE_SCAN_DIALOG_SHOW": "MERGE_SCAN_DIALOG_SHOW",
    "PAUSE_SCAN": "PAUSE_SCAN",
    "RESET_SCAN_ADD_STATUS": "RESET_SCAN_ADD_STATUS",
    "RESTART_SCAN": "RESTART_SCAN",
    "START_SCAN": "START_SCAN",
    "UPDATE_SCAN": "UPDATE_SCAN",
  },
  "sourcesTypes": Object {
    "ADD_SOURCE": "ADD_SOURCE",
    "CREATE_SOURCE_SHOW": "CREATE_SOURCE_SHOW",
    "DELETE_SOURCE": "DELETE_SOURCE",
    "DELETE_SOURCES": "DELETES_SOURCE",
    "EDIT_SOURCE_SHOW": "EDIT_SOURCE_SHOW",
    "GET_SCANS_SOURCES": "GET_SCANS_SOURCES",
    "GET_SOURCE": "GET_SOURCE",
    "GET_SOURCES": "GET_SOURCES",
    "INVALID_SOURCE_WIZARD_STEPTWO": "INVALID_SOURCE_WIZARD_STEPTWO",
    "UPDATE_SOURCE": "UPDATE_SOURCE",
    "UPDATE_SOURCE_HIDE": "UPDATE_SOURCE_HIDE",
    "UPDATE_SOURCE_WIZARD": "UPDATE_SOURCE_WIZARD",
    "UPDATE_SOURCE_WIZARD_STEPONE": "UPDATE_SOURCE_WIZARD_STEPONE",
    "UPDATE_SOURCE_WIZARD_STEPTWO": "UPDATE_SOURCE_WIZARD_STEPTWO",
  },
  "statusTypes": Object {
    "STATUS_INFO": "STATUS_INFO",
    "default": "STATUS_INFO",
  },
  "toastNotificationTypes": Object {
    "TOAST_ADD": "TOAST_ADD",
    "TOAST_PAUSE": "TOAST_PAUSE",
    "TOAST_REMOVE": "TOAST_REMOVE",
    "TOAST_RESUME": "TOAST_RESUME",
  },
  "userTypes": Object {
    "USER_AUTH": "USER_AUTH",
    "USER_INFO": "USER_INFO",
    "USER_LOGOUT": "USER_LOGOUT",
  },
  "viewPaginationTypes": Object {
    "SET_PER_PAGE": "SET_PER_PAGE",
    "VIEW_FIRST_PAGE": "VIEW_FIRST_PAGE",
    "VIEW_LAST_PAGE": "VIEW_LAST_PAGE",
    "VIEW_NEXT_PAGE": "VIEW_NEXT_PAGE",
    "VIEW_PAGE_NUMBER": "VIEW_PAGE_NUMBER",
    "VIEW_PREVIOUS_PAGE": "VIEW_PREVIOUS_PAGE",
  },
  "viewToolbarTypes": Object {
    "ADD_FILTER": "ADD_FILTER",
    "CLEAR_FILTERS": "CLEAR_FILTERS",
    "REMOVE_FILTER": "REMOVE_FILTER",
    "SET_FILTER_TYPE": "SET_FILTER_TYPE",
    "SET_FILTER_VALUE": "SET_FILTER_VALUE",
    "SET_SORT_TYPE": "SET_SORT_TYPE",
    "TOGGLE_SORT_ASCENDING": "TOGGLE_SORT_ASCENDING",
  },
  "viewTypes": Object {
    "CREDENTIALS_VIEW": "CREDENTIALS_VIEW",
    "DESELECT_ITEM": "DESELECT_ITEM",
    "EXPAND_ITEM": "EXPAND_ITEM",
    "SCANS_VIEW": "SCANS_VIEW",
    "SELECT_ITEM": "SELECT_ITEM",
    "SOURCES_VIEW": "SOURCES_VIEW",
  },
}
`;

exports[`specific types 1`] = `
Object {
  "confirmationModal": Object {
    "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
    "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
  },
  "credentials": Object {
    "ADD_CREDENTIAL": "ADD_CREDENTIAL",
    "CREATE_CREDENTIAL_SHOW": "CREATE_CREDENTIAL_SHOW",
    "DELETE_CREDENTIAL": "DELETE_CREDENTIAL",
    "DELETE_CREDENTIALS": "DELETE_CREDENTIALS",
    "EDIT_CREDENTIAL_SHOW": "EDIT_CREDENTIAL_SHOW",
    "GET_CREDENTIAL": "GET_CREDENTIAL",
    "GET_CREDENTIALS": "GET_CREDENTIALS",
    "GET_WIZARD_CREDENTIALS": "GET_WIZARD_CREDENTIALS",
    "RESET_CREDENTIAL_UPDATE_STATUS": "RESET_CREDENTIAL_UPDATE_STATUS",
    "UPDATE_CREDENTIAL": "UPDATE_CREDENTIAL",
    "UPDATE_CREDENTIAL_HIDE": "UPDATE_CREDENTIAL_HIDE",
  },
  "facts": Object {
    "ADD_FACTS": "ADD_FACTS",
    "default": "ADD_FACTS",
  },
  "reports": Object {
    "GET_MERGE_REPORT": "GET_MERGE_REPORT",
    "GET_REPORT": "GET_REPORT",
  },
  "scans": Object {
    "ADD_SCAN": "ADD_SCAN",
    "CANCEL_SCAN": "CANCEL_SCAN",
    "DELETE_SCAN": "DELETE_SCAN",
    "GET_MERGE_SCAN_RESULTS": "GET_MERGE_SCAN_RESULTS",
    "GET_SCAN": "GET_SCAN",
    "GET_SCANS": "GET_SCANS",
    "GET_SCAN_CONNECTION_RESULTS": "GET_SCAN_CONNECTION_RESULTS",
    "GET_SCAN_INSPECTION_RESULTS": "GET_SCAN_INSPECTION_RESULTS",
    "GET_SCAN_JOB": "GET_SCAN_JOB",
    "GET_SCAN_JOBS": "GET_SCAN_JOBS",
    "MERGE_SCAN_DIALOG_HIDE": "MERGE_SCAN_DIALOG_HIDE",
    "MERGE_SCAN_DIALOG_SHOW": "MERGE_SCAN_DIALOG_SHOW",
    "PAUSE_SCAN": "PAUSE_SCAN",
    "RESET_SCAN_ADD_STATUS": "RESET_SCAN_ADD_STATUS",
    "RESTART_SCAN": "RESTART_SCAN",
    "START_SCAN": "START_SCAN",
    "UPDATE_SCAN": "UPDATE_SCAN",
  },
  "sources": Object {
    "ADD_SOURCE": "ADD_SOURCE",
    "CREATE_SOURCE_SHOW": "CREATE_SOURCE_SHOW",
    "DELETE_SOURCE": "DELETE_SOURCE",
    "DELETE_SOURCES": "DELETES_SOURCE",
    "EDIT_SOURCE_SHOW": "EDIT_SOURCE_SHOW",
    "GET_SCANS_SOURCES": "GET_SCANS_SOURCES",
    "GET_SOURCE": "GET_SOURCE",
    "GET_SOURCES": "GET_SOURCES",
    "INVALID_SOURCE_WIZARD_STEPTWO": "INVALID_SOURCE_WIZARD_STEPTWO",
    "UPDATE_SOURCE": "UPDATE_SOURCE",
    "UPDATE_SOURCE_HIDE": "UPDATE_SOURCE_HIDE",
    "UPDATE_SOURCE_WIZARD": "UPDATE_SOURCE_WIZARD",
    "UPDATE_SOURCE_WIZARD_STEPONE": "UPDATE_SOURCE_WIZARD_STEPONE",
    "UPDATE_SOURCE_WIZARD_STEPTWO": "UPDATE_SOURCE_WIZARD_STEPTWO",
  },
  "status": Object {
    "STATUS_INFO": "STATUS_INFO",
    "default": "STATUS_INFO",
  },
  "toastNotifications": Object {
    "TOAST_ADD": "TOAST_ADD",
    "TOAST_PAUSE": "TOAST_PAUSE",
    "TOAST_REMOVE": "TOAST_REMOVE",
    "TOAST_RESUME": "TOAST_RESUME",
  },
  "user": Object {
    "USER_AUTH": "USER_AUTH",
    "USER_INFO": "USER_INFO",
    "USER_LOGOUT": "USER_LOGOUT",
  },
  "view": Object {
    "CREDENTIALS_VIEW": "CREDENTIALS_VIEW",
    "DESELECT_ITEM": "DESELECT_ITEM",
    "EXPAND_ITEM": "EXPAND_ITEM",
    "SCANS_VIEW": "SCANS_VIEW",
    "SELECT_ITEM": "SELECT_ITEM",
    "SOURCES_VIEW": "SOURCES_VIEW",
  },
  "viewPagination": Object {
    "SET_PER_PAGE": "SET_PER_PAGE",
    "VIEW_FIRST_PAGE": "VIEW_FIRST_PAGE",
    "VIEW_LAST_PAGE": "VIEW_LAST_PAGE",
    "VIEW_NEXT_PAGE": "VIEW_NEXT_PAGE",
    "VIEW_PAGE_NUMBER": "VIEW_PAGE_NUMBER",
    "VIEW_PREVIOUS_PAGE": "VIEW_PREVIOUS_PAGE",
  },
  "viewToolbar": Object {
    "ADD_FILTER": "ADD_FILTER",
    "CLEAR_FILTERS": "CLEAR_FILTERS",
    "REMOVE_FILTER": "REMOVE_FILTER",
    "SET_FILTER_TYPE": "SET_FILTER_TYPE",
    "SET_FILTER_VALUE": "SET_FILTER_VALUE",
    "SET_SORT_TYPE": "SET_SORT_TYPE",
    "TOGGLE_SORT_ASCENDING": "TOGGLE_SORT_ASCENDING",
  },
}
`;
